alpha-core/src/test/java/at/ac/tuwien/kr/alpha/core/solver/TrailAssignmentTest.java

```diff
@@ -180,8 +181,8 @@ public void newAssignmentsIteratorAndBacktracking() {
 
 		newAssignmentsIterator = assignment.getNewPositiveAssignmentsIterator();
 
-		assertEquals(1, newAssignmentsIterator.next());
-		assertEquals(2, newAssignmentsIterator.next());
+		assertEquals(1, (int)newAssignmentsIterator.next());
+		assertEquals(2, (int)newAssignmentsIterator.next());
```

Comment:
Please revert all changes to this file, it re-introduces compiler warnings that were fixed in master. It seems no other changes (besides the re-shuffling of import statements) were made, and the file could be reverted completely.